apps/seqan_tcoffee/seqan_tcoffee.cpp

```diff
@@ -24,6 +24,8 @@ Lesser General Public License for more details.
 
 using namespace seqan;
 
+//#define SEQAN_DEBUG
+
```

Comment:
You should use your personal debug flag if you want to print debug messages or measure time. 
The ```SEQAN_DEBUG``` flag is used to enable assertions and such kind of things and has a global usage.